.gitignore

```diff
@@ -161,3 +161,7 @@ cython_debug/
 
 # Documentation
 **/_autosummary
+
+# anaconda related directories
```

Comment:
We can also remove it. I put it there because I use a vscode extension that creates these folders in the project folder, thus they always show up in git status etc.
I am pretty sure we will never want to add a .conda environment to git, so adding it in the .gitignore along all the other ignored stuff will not hurt.

Reply:
Our .gitignore is really big anyway so probably doesn't matter. I prefer to use something like .*/ to e.g. ignore all folders starting with a period and then adding specific folders which are needed. Makes it easier to read and maintain. Maybe for the future...